src/js/jsx/sections/layers/LayerGroup.jsx

```diff
@@ -116,7 +136,7 @@ define(function (require, exports, module) {
                                 disabled={this.props.disabled}
                                 document={this.props.document}
                                 layer={layer}
-                                childNodes={layerNode.children}/>
+                                hasChildren={!!layerNode.children}/>
```

Comment:
Eh, it may not actually be better, but you can do it with the `LayerStructure` instance: `document.layers.children(layer)`.